clang/lib/Sema/CheckExprLifetime.cpp

```diff
@@ -966,7 +969,8 @@ static bool pathOnlyInitializesGslPointer(IndirectLocalPath &Path) {
     if (It.Kind == IndirectLocalPathEntry::LifetimeBoundCall)
       continue;
     return It.Kind == IndirectLocalPathEntry::GslPointerInit ||
-           It.Kind == IndirectLocalPathEntry::GslReferenceInit;
+           It.Kind == IndirectLocalPathEntry::GslReferenceInit ||
```

Comment:
nit: rename this function to `pathOnlyHandlesGslPointer` as it is not specific to initialisation.

Reply:
Done.